Gopkg.lock

```diff
@@ -29,14 +23,20 @@
   branch = "master"
   name = "github.com/cloudfoundry/socks5-proxy"
   packages = ["."]
-  revision = "6afb711f4b32a266b51160288e98fd3c4b28f0cb"
+  revision = "a5dbd34501d246ab6c004c9afebb5bd0dd9d7d22"
 
 [[projects]]
   name = "github.com/davecgh/go-spew"
   packages = ["spew"]
   revision = "346938d642f2ec3594ed81d874461961cd0faa76"
   version = "v1.1.0"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/genevievelesperance/go-socks5"
+  packages = ["."]
+  revision = "f07e0bb84629ba30f60da69274cd1d9dbb4a1d08"
```

Comment:
Does cloudfoundry/socks5-proxy introduce a dependency on @genevievelesperance’s personal repository? Is there a reason it couldn’t be merged into cloudfoundry/go-socks5?

Reply:
Hi @dpb587, this does introduce a dependency on a personal fork, however there is no cloudfoundry/go-socks5. This is a fork of https://github.com/armon/go-socks5 and a pull request has been open since December to merge the changes found in genevieve's fork. https://github.com/armon/go-socks5/pull/21. The source repo doesn't appear to be well maintained as no changes have been made or accepted since 2016.

This is not a direct dependency of bosh-utils, but a transitive dependency through cloudfoundry/socks5-proxy: https://github.com/cloudfoundry/socks5-proxy/blob/master/Gopkg.lock#L4-L8 so this decision appears to have already been made by the team who maintain that repo.